waterlog/src/Components/DailyWastage.js

```diff
@@ -1,34 +1,53 @@
 import React from 'react';
 import { Line, defaults } from 'react-chartjs-2';
+import { Globals } from './../Globals';
 
 const DailyWastageComponent = (props) => {
+  let labels = props.props.dataPoints.map(a => (new Date(a.x).getHours()+":00"));
+  let datas = props.props.dataPoints.map(a => a.y);
```

Comment:
'datas'? Funny variable name, why not something like dataY since you're taking the Y out of the dataPoints, whatever y is.

Reply:
Renamed to dataY. Thanks for the feedback.

Reply:
Nice.